ui/src/components/flows/FlowRun.vue

```diff
@@ -112,6 +112,21 @@
                     input.focus()
                 }
             }, 500)
+
+            this._keyListener = function(e) {
+                if (e.keyCode === 13) {
+                    if(e.shiftKey){
+                        return true;
+                    }
+                    e.preventDefault()
+                    this.onSubmit(this.$refs.form);
+                }
+            };
+
+            document.addEventListener("keydown", this._keyListener.bind(this));
```

Comment:
We can't find a proper lib for that to be able to add shortcut on the whole application quickly (and don't care about the remove listeners leak ?